RELEASE-NOTES.txt

```diff
@@ -3,10 +3,13 @@
 * [*] [Jetpack-only] Reader: Fix the display of the followed topics in the filter sheet when they're written in non-latin languages. [#20702]
 * [*] [Jetpack-only] Reader: Fixed an issue that was causing the Report and Block actions to be missing from Post Menu actions sheet. [#20705]
 * [***] Enables editing of the site homepage for sites using block-based themes directly from the pages list. [#20713]
+* [*] Add disabled style to `Cell` component [https://github.com/WordPress/gutenberg/pull/50665]
+* [**] Fix undo/redo history when inserting a link configured to open in a new tab [https://github.com/WordPress/gutenberg/pull/50460]
+* [**] Disable details settings for not belonged VideoPress videos [https://github.com/wordpress-mobile/gutenberg-mobile/pull/5782]
```

Comment:
This change will be only available in the Jetpack app because the VideoPress block is a Jetpack-only feature. In this case, we usually add the prefix `[Jetpack-only]` to this type of change so they are not included in the what's new of the WordPress app.